tests/robot-tests/tests/admin_and_public/bau/publish_data.robot

```diff
@@ -495,6 +495,29 @@ Add headline text block to Content page again
     user adds headlines text block
     user adds content to headlines text block    Headline text block text
 
+Add an accordion section to release content
+    user clicks button    Add new section
+    user changes accordion section title    1    Test section
+
+Add text block with link to a featured table to accordion section
+    user adds text block to editable accordion section    Test section    id:releaseMainContent
+    ${block}=    user starts editing accordion section text block    Test section    1    id:releaseMainContent
+    ${toolbar}=    get editor toolbar    ${block}
+    ${insert}=    user gets button element    Insert    ${toolbar}
+    user clicks element    ${insert}
+    ${button}=    user gets button element    Insert featured table link    ${toolbar}
+    user clicks element    ${button}
+    ${modal}=    user waits until modal is visible    Insert featured table link
+    user enters text into element    id:featuredTablesSearch-input    tes
+    user waits until page contains element    id:featuredTablesSearch-options
+    user clicks element    id:featuredTablesSearch-option-0
+    user clicks button    Insert    ${modal}
+    user waits until modal is not visible    Insert featured table link
+    user clicks button    Save & close
```

Comment:
Again, can probably use "user saves autosaving block blah blah" rather than:

```
    user clicks button Save & close
    user waits until page finishes loading
    user waits until parent contains button    ${block}    Edit block
```

Reply:
I've changed it to use `user saves autosaving text block` now.

Reply:
Brill thanks :slightly_smiling_face: